core/modules/layout/tests/layout.test

```diff
@@ -50,6 +50,35 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
   /**
    * Add and remove blocks from a custom layout path.
    */
+  function testLayoutTemplates() {
+    // Attempt to disable the 1 column layout (should fail).
+    $this->backdropGet('admin/structure/layouts/settings');
+    $edit = array(
+      'templates[one_column]' => FALSE,
+      'templates[two_column]' => TRUE,
+      'templates[two_column_flipped]' => TRUE,
+      'templates[three_three_four_column]' => TRUE,
+    );
+    $this->backdropPost(NULL, $edit, t('Save configuration'));
+    $this->assertText(t('The 1 column layout template is currently in use and may not be excluded.'));
```

Comment:
Per above, the message should wrap the name of the template in quotes, e.g. `The "1 column" layout template...`